src/wait-for-peers.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
 
+let interval = null
+
```

Comment:
is there any issue with this variable being reassigned by another waitForPeers call, overwriting the first interval and making it unable to be cleared?

Reply:
@tabcat It might be a problem, I've just noticed that someone already proposed solution to that problem in PR https://github.com/ipfs-shipyard/ipfs-pubsub-1on1/pull/10. If for some reason it's not a solution let me know and I'll try to find a solution